### PR TITLE
Fixes several issues.

### PR DESCRIPTION
**Pull request by sfshaza2**
staged:  https://sz-flutter.firebaseapp.com

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -51,7 +51,10 @@ <h1 class="homepage__hero__text">Publish <span class="color-white">mobile, web,
             <div class="col-md-12 col-lg-8 offset-lg-2">
                 <span class="homepage__intro__partner">Made by {% asset homepage/logo-google.svg alt='Google logo' %}</span>
                 <p class="homepage__intro__statement text-center">
-                    Flutter is Google’s portable UI toolkit for building beautiful, natively-compiled applications for <a href="https://flutter.dev/docs">mobile</a>, <a href="https://flutter.dev/web">web</a>, and <a href="https://flutter.dev/desktop">desktop</a> from a single codebase.
+                    Flutter is Google’s portable UI toolkit for building
```
Comment (legalcodes) [inline src/index.html]: Here's some more center-aligned p text. Do we intentionally do this in some cases? 

Screen Shot 2019-06-14 at 2 13 19 PM

File: src/index.html [hunk 2]
```diff
@@ -60,7 +63,8 @@ <h1 class="homepage__hero__text">Publish <span class="color-white">mobile, web,
                 </div>
 
                 <p>
-                    Coming from another platform? Docs: <a href="/docs/get-started/flutter-for/ios-devs">iOS</a>,
+                    Coming from another platform? Docs:
```
Comment (legalcodes) [inline src/index.html]: `Docs:`
Sentence fragment

File: src/index.html [hunk 3]
```diff
@@ -81,7 +85,8 @@ <h1 class="homepage__hero__text">Publish <span class="color-white">mobile, web,
                 <h3>Fast Development</h3>
 
                 <p>
-                Paint your app to life in milliseconds with stateful Hot Reload. Use a rich set of fully-customizable
+                Paint your app to life in milliseconds with stateful
+                Hot Reload. Use a rich set of fully-customizable
```
Comment (legalcodes) [inline src/index.html]: `with stateful Hot Reload`

Our [hot reload doc](https://flutter.dev/docs/development/tools/hot-reload) uses all lowercase i.e. `stateful hot reload`. Should we pick one and go with it?

File: src/community/index.html [hunk 4]
```diff
@@ -59,7 +59,8 @@ <h3>{{ case.title }}</h3>
             <div class="col-lg-6 offset-lg-3">
                 <h2>Join the conversation</h2>
                 <p>
-                    Get answers to your questions and stay up to date with everything new and exciting happening in the
+                    Get answers to your questions and stay up to date
```
[the remark this answers is not in the document]
Comment (legalcodes) [reply src/community/index.html]: Or should it? We seem to do this elsewhere. Do we have a convention for this?

File: src/index.html (hunk 1 above)
Comment (sfshaza2) [reply src/index.html]: Definitely. This page was designed by a UX designer with Martin's input.

File: src/index.html (hunk 2 above)
Comment (sfshaza2) [reply src/index.html]: Yeah, again, UX Designer and Martin.  :)

File: src/index.html (hunk 3 above)
Comment (sfshaza2) [reply src/index.html]: Actually, if you look at go/dart-terms, it's either hot reload (lower case) or Stateful Hot Reload, according to that doc. I believe that came from marketing or the PMs.

File: src/community/index.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (sfshaza2) [reply src/community/index.html]: Yes, it should.  :D

**Pull request merged**